src/components/BuildConfigRestoreModal/BuildConfigRestoreModalButton.tsx

```diff
@@ -4,12 +4,15 @@ import { ProtectedComponent } from 'components/ProtectedContent/ProtectedCompone
 
 interface IBuildConfigRestoreModalButtonProps {
   toggleModal: () => void;
+  isDisabled?: boolean;
 }
 
-export const BuildConfigRestoreModalButton = ({ toggleModal }: IBuildConfigRestoreModalButtonProps) => (
+export const BuildConfigRestoreModalButton = ({ toggleModal, isDisabled = false }: IBuildConfigRestoreModalButtonProps) => (
   <ProtectedComponent>
-    <Button variant="primary" onClick={toggleModal} size="sm">
-      Restore
-    </Button>
+    <div title={isDisabled ? 'Restore button is disabled for the current revision' : undefined}>
```

Comment:
For disabled tooltips we use slightly different syntax, see for example https://github.com/project-ncl/pnc-web-ui-react/blob/eb23ce3ea54a65cbed9ac8bfe6c7f718171e52ae/src/components/BrewPushModal/BrewPushModalButton.tsx#L18